brave/brave-core
# Simplify Android Report Broken Site language and point to external link

Resolves https://github.com/brave/brave-browser/issues/36483



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Open Brave Shields pop up dialog.
2. Switch off Brave Shields if It is ON.
3. Click "Report broken site".
4. Make sure that It contains the text mentioned here: https://github.com/brave/brave-browser/issues/35683#issuecomment-1917947647, and "`Learn More.`" - part is clickable external link to the Webcompat wiki page
